utils/usb/__init__.py

```diff
@@ -2,12 +2,12 @@
 
 import sys
 #from main import log
-from ..model_lab.logger import log
+from ..splitter.logger import log
 
 def init_usbmon() -> bool:
     import os
     dirs = os.listdir("/dev/")
-    if "usbmon0" in dirs:
+    if "usbmon4" in dirs:
```

Comment:
should this maybe be passed as an argument in case it varies across systems?

Reply:
yes, would be much better

Reply:
it varies also across different runs on the same system. if the system reboots, it will surely change.